spring-cloud-aws-testcontainers/src/test/java/io/awspring/cloud/testcontainers/AwsContainerConnectionDetailsFactoryTest.java

```diff
@@ -88,6 +93,14 @@ void configuresS3ClientWithServiceConnection(@Autowired S3Client client) {
 		assertThatCode(client::listBuckets).doesNotThrowAnyException();
 	}
 
+	@Test
+	@Disabled
+	void configuresS3PresignerWithServiceConnection(@Autowired S3Presigner s3Presigner) {
```

Comment:
@maciejwalkowiak @MatejNedic 
I have disabled this test because I am uncertain how to verify that the `S3Presigner` correctly references the endpoint property from the `ServiceConnection`. I noticed that most client tests simply call one of the basic methods and ensure that the call does not result in an exception.

In this case, I attempted to use the` presignGetObject` method for verification. However, this method requires several parameters to be set up, and it might also necessitate configuring an S3 bucket prior to running the test. This has left me uncertain whether my approach is correct. I would appreciate your opinion or guidance on how to proceed with this.

Reply:
@maciejwalkowiak @MatejNedic @tomazfernandes 
I have fixed the test, please let me know if I have validated test correctly.

Reply:
@maciejwalkowiak @MatejNedic @tomazfernandes 
Is there any way to retry the pipeline, because the test which is failing in the CI process is passing locally.